app-android/src/main/java/com/ustadmobile/port/android/view/SiteEditFragment.kt

```diff
@@ -128,4 +155,124 @@ class SiteEditFragment: UstadEditFragment<Site>(), SiteEditView {
 
     }
 
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun SiteEditScreen(
+    uiState: SiteEditUiState,
+    onSiteChanged: (Site?) -> Unit = {},
+    onItemClicked: (SiteTermsWithLanguage) -> Unit = {},
+    onDeleteIconClicked: (SiteTermsWithLanguage) -> Unit = {}
+){
+    Column (
+        modifier = Modifier
+            .padding(8.dp)
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.Start
+    ){
+
+        UstadTextEditField(
+            value = uiState.site?.siteName ?: "",
+            label = stringResource(id = R.string.name),
+            error = uiState.siteNameError,
+            enabled = uiState.fieldsEnabled,
+            onValueChange = {
+                onSiteChanged(uiState.site?.shallowCopy{
+                    siteName = it
+                })
+            }
+        )
+
+        CustomSwitch(
+            checked = uiState.site?.guestLogin ?: false,
+            label = stringResource(R.string.guest_login_enabled),
+            onSwitchChanged = {
+                onSiteChanged(uiState.site?.shallowCopy{
+                    guestLogin = it
+                })
+            }
+        )
+
+        CustomSwitch(
+            checked = uiState.site?.registrationAllowed ?: false,
+            label = stringResource(R.string.registration_allowed),
+            onSwitchChanged = {
+                onSiteChanged(uiState.site?.shallowCopy{
+                    registrationAllowed = it
+                })
+            }
+        )
+
+        Text(
+            stringResource(R.string.terms_and_policies),
+            style = Typography.h6
+        )
+
```

Comment:
On the edit screen we need the option to add an item. The add item is itself a ListItem. It should have an onClick function (e.g. onClickNewTerms). 

Like other lists on Android, the add item option is itself a listItem. It should have an add icon. ListItem has an option for the main icon. 
e.g. 
```
ListItem(
  ..
  icon = Icon(
                 imageVector = Icons.Filled.Add,
                 contentDescription = null
)
Note: content description is null because the icon is just for decoration, and there is text for adding a new item. It is not like the delete button, where there is no text.

```